examples/observability/eks-cluster-with-adot-amp-grafana-for-haproxy/README.md

```diff
@@ -0,0 +1,223 @@
+applications. However, you can follow the steps below to build and deploy and example
+application.
+
+---
+
+#### Verify that the Resources Deployed Successfully
+
+- Verify that Amazon Managed Prometheus workspace was created successfully:
+
+  - Check the status of Amazon Managed Prometheus workspace through the AWS console.
+
+- Check that OpenTelemetry Collector is running successfully inside EKS:
+
+```
+kubectl get pods -A
+
+NAMESPACE                       NAME                                                         READY   STATUS    RESTARTS   AGE
+cert-manager                    cert-manager-7989877dff-jxk57                                1/1     Running   0          160m
+cert-manager                    cert-manager-cainjector-7d55bf8f78-jcc6d                     1/1     Running   0          160m
+cert-manager                    cert-manager-webhook-577f77586f-m6mlg                        1/1     Running   0          160m
+kube-system                     aws-node-kvbdl                                               1/1     Running   0          3h36m
+kube-system                     aws-node-lv4g4                                               1/1     Running   0          3h36m
+kube-system                     aws-node-x8zcs                                               1/1     Running   0          3h36m
+kube-system                     coredns-745979c988-bhtx6                                     1/1     Running   0          3h42m
+kube-system                     coredns-745979c988-ktdlg                                     1/1     Running   0          3h42m
+kube-system                     kube-proxy-2wqr2                                             1/1     Running   0          3h36m
+kube-system                     kube-proxy-7kz4p                                             1/1     Running   0          3h36m
+kube-system                     kube-proxy-rxkp8                                             1/1     Running   0          3h36m
+opentelemetry-operator-system   adot-collector-64c8b46888-q6s98                              1/1     Running   0          158m
+opentelemetry-operator-system   opentelemetry-operator-controller-manager-68f5b47944-pv6x7   2/2     Running   0          158m
+```
+
+- Open your Managed Grafana Workspace, head to the configuration page and and verify that Amazon Managed Prometheus was added as a default data source, test its connectivity.
+
+#### Deploy an Example Application
+
+In this section we will deploy sample application and extract metrics using  AWS OpenTelemetry collector
+
+- 1. Add the helm incubator repo:
+```
+helm repo add haproxy-ingress https://haproxy-ingress.github.io/charts
+```
+
+- 2. Enter the following command to create a new namespace:
+```
+kubectl create namespace haproxy-ingress-sample
+```
+
+- 3. Enter the following commands to install HAProxy:
+```
+helm install haproxy haproxy-ingress/haproxy-ingress \
```

Comment:
Not required now - but since this is an example to get people started quickly, we can Terraform this helm release and cut down on the number of manual kubectl steps users need to take to get this live

Reply:
Yeah.. we could.. bt the customers are not obliged to install them.. that's the reason we kept them on the README to help customers verify the working